ci-operator/step-registry/ovn/validate/ovn-validate-commands.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+set -o errexit
+set -o nounset
+set -o pipefail
+
+abortJob=false
+EXPECTED_MASTER_POD_COUNT=3
```

Comment:
In CI is the 3/6 number always going to hold true?

Reply:
yeah, I mentioned that I was worried about that in a [comment](https://github.com/openshift/release/pull/18668#issuecomment-845358863)

we can expose these variables all the way up to the top level job definition if we need to tweak them per job